src/doc/src/commands/cargo-miri.md

```diff
@@ -0,0 +1,20 @@
+# cargo-miri(1)
+
+## NAME
+
+cargo-miri --- Runs binary crates and tests in Miri
+
+## DESCRIPTION
+
+This is an external command distributed with the Rust toolchain as an optional component.
+It is not built into Cargo, and may require additional installation.
+
+For information about usage and installation,
+see <https://github.com/rust-lang/miri>.
```

Comment:
> Do we really want to document the nightly-only cargo miri? 

When we have unstable options, we mark them. Unsure about unstable commands.  As this is a nightly command, we should probably mark it as unstable